mesonbuild/modules/python.py

```diff
@@ -94,14 +94,46 @@ def __init__(self, python_holder, environment, kwargs):
         else:
             self.pkgdep = None
 
-            if mesonlib.is_windows() and DependencyMethods.SYSCONFIG in self.methods:
-                self._find_libpy_windows(environment)
+            if DependencyMethods.SYSCONFIG in self.methods:
+                if mesonlib.is_windows():
+                    self._find_libpy_windows(environment)
+                else:
+                    self._find_libpy(python_holder, environment)
 
         if self.is_found:
             mlog.log('Dependency', mlog.bold(self.name), 'found:', mlog.green('YES'))
         else:
             mlog.log('Dependency', mlog.bold(self.name), 'found:', mlog.red('NO'))
 
+    def _find_libpy(self, python_holder, environment):
+        if python_holder.is_pypy:
+            if self.major_version == 3:
+                libname = 'pypy3-c'
+            else:
+                libname = 'pypy-c'
+            libdir = os.path.join(self.variables.get('base'), 'bin')
+            libdirs = [libdir]
+        else:
+            libname = 'python{}'.format(self.version)
+            if 'DEBUG_EXT' in self.variables:
+                libname += self.variables['DEBUG_EXT']
+            if 'ABIFLAGS' in self.variables:
+                libname += self.variables['ABIFLAGS']
+            libdirs = []
+
+        largs = self.compiler.find_library(libname, environment, libdirs)
+
+        self.is_found = largs != None
```

Comment:
__[Flake8]__

[E711] comparison to None should be 'if cond is not None:'


<!-- Comment by [SideCI](https://sideci.com) -->